src/docker.go

```
@@ -1,676 +1,676 @@
+		return
+	}
+	defer reader.Close()
+
+	// 设置响应头
+	c.Header("Content-Type", "application/octet-stream")
+	c.Header("Content-Length", fmt.Sprintf("%d", size))
+	c.Header("Docker-Content-Digest", digest)
+
+	// 流式传输blob内容
+	c.Status(http.StatusOK)
+	io.Copy(c.Writer, reader)
```

golangci-lint (1.64.8)

Error return value of `io.Copy` is not checked

(errcheck)